src/main.cpp

```diff
@@ -39,16 +39,16 @@
 const char* const short_options = "vhw:e:dp:";
 
 const struct option long_options[] = {
-    {"version", 0, NULL, 'v'},
-    {"help",    0, NULL, 'h'},
-    {"workdir", 1, NULL, 'w'},
-    {"execute", 1, NULL, 'e'},
-    {"drop",    0, NULL, 'd'},
-    {"profile", 1, NULL, 'p'},
-    {NULL,      0, NULL,  0}
+    {"version", 0, nullptr, 'v'},
+    {"help",    0, nullptr, 'h'},
+    {"workdir", 1, nullptr, 'w'},
+    {"execute", 1, nullptr, 'e'},
+    {"drop",    0, nullptr, 'd'},
+    {"profile", 1, nullptr, 'p'},
+    {nullptr,      0, nullptr,  0}
```

Comment:
Minor style issue: this line is no longer aligned with previous lines.

Reply:
Nice catch. Will fix it.